src/Microsoft.ML.StandardTrainers/Standard/Online/OnlineGradientDescent.cs

```diff
@@ -23,8 +23,17 @@
 
 namespace Microsoft.ML.Trainers
 {
-
-    /// <include file='doc.xml' path='doc/members/member[@name="OGD"]/*' />
+    /// <summary>
+    /// The <see cref="IEstimator{TTransformer}"/> for training a linear regression model using
+    /// online gradient descent (OGD) for estimating the parameters of the linear regression model.
```

Comment:
> online gradient descent [](start = 7, length = 24)

feel like those are usually capitalized. 

see here: [https://papers.nips.cc/paper/3319-adaptive-online-gradient-descent.pdf](https://papers.nips.cc/paper/3319-adaptive-online-gradient-descent.pdf)

Reply:
You're looking at Title Case in the paper. There are all-lowercase instances in that same paper. But I changed it anyways.

---
In reply to: [268038471](https://github.com/dotnet/machinelearning/pull/3059#discussion_r268038471) [](ancestors = 268038471)